draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -221,45 +221,37 @@
     </section>
     
     <section title="QoS Model Design">
-      <t>A classifier consists of packets which may be grouped when a
-      logical set of rules are applied on different packet header
-      fields. The grouping may be based on different values or range
-      of values of same packet header field, presence or absence of
-      some values or range of values of a packet field or a
-      combination thereof. The QoS classifier is defined in the
+      <t> A classifier consists of set of rules which selects a 
+      group of packets when the rules are applied to different
+      different packet header. The rules may be based on any
+      combination of values or range of values of different
+      packet header field. The QoS classifier is defined in the
```

Comment:
How about this?

A classifier consists of a filter and an operation that is performed on that filter. The filter defines a set of rules that dictate how the packet will be classified, e.g. whether it will be classified based on the DSCP marking, or will be classified based on source IPv4 address prefix. The operation defines whether these classifications are logically AND or OR. The filters may be based on a combination of values or range of values of different parts of the packet header field. The QoS classifier is defined in the ...

Reply:
we call "a match criteria" as a filter. So, a classifier will be set of filters and a logical operation that defines how the filters will be combined. There is rfc5777 section 4 which vaguely defines classifier. or we can modify the text as:  A classifier consists of set of filters and an operation that is performed on those filters. Each filter defines a set of rules on a particular packet header field that dictate how the packet will be classified based on that particular header field, e.g. whether it will be classified based on the DSCP marking, or will be classified based on source IPv4 address prefix. The operation defines whether these filters are logically AND or OR. The filters may be based on a combination of values or range of values of different parts of the packet header field. The QoS classifier is defined in the...